v2.1/admin-ui-user-authentication.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Secure the Admin UI
+summary: Learn how to enable user authentication for secure clusters for the Admin UI.
+toc: false
+---
+
+By default, CockroachDB allows all users to access and view the Admin UI. However, for secure clusters, you can enable user authentication so that only authorized users can access and view the Admin UI.
+
+<div id="toc"></div>
+
+1. Start a secure cluster as described in our [deployment tutorials](secure-a-cluster.html#step-1-create-security-certificates).
```

Comment:
This isn't the right link. It should be `manual-deployment.html`. From there, whatever deployment guide they choose will default to the secure version.